Allwmake

```diff
@@ -71,7 +71,7 @@ else
     if [ -d ${OPENFAST_DIR} ]
     then
         echo "OPENFAST_DIR is set to '${OPENFAST_DIR}'. Attempting to compile OpenFAST-coupled solver."
-        wmake applications/solvers/superDeliciousVanilla.turbine
+       #wmake applications/solvers/superDeliciousVanilla.turbine
```

Comment:
@mchurchf did you mean to leave this commented out?

Reply:
The `.turbine` solver is now obsolete. The regular `superDeliciousVanilla` handles turbine and no-turbine cases. Caveat: needs to be compiled with OpenFAST coupling even if you don't plan to use turbines.

Reply:
Fixed by [b7b82cd](https://github.com/NREL/SOWFA-6/pull/50/commits/b7b82cd4a958f14a3f4e6b5c9a692b4757bd901b)